nim-compile.el

```diff
@@ -91,33 +78,25 @@ The config file would one of those: config.nims, PROJECT.nim.cfg, or nim.cfg."
 
 (defun nim-compile--set-compile-command ()
   "Set ‘compile-command’ for Nim language."
-  (let ((file (when buffer-file-name
-                (shell-quote-argument buffer-file-name)))
-        cmd)
-    (when file
-      (setq cmd
-            (cond
-             ((eq 'nimscript-mode major-mode)
-              (let ((pfile (nim-get-project-file '(".nims" ".nimble"))))
-                (cond
-                 ;; as build tool
-                 ((nim-nimble-file-p file)
-                  (let ((nim-compile-command "nimble"))
-                    (nim--fmt '("build") "")))
-                 ((and (nim-nims-file-p pfile)
-                       (equal pfile buffer-file-name))
-                  (nim--fmt '("build") pfile))
-                 (t
-                  ;; as script file
-                  (nim--fmt '("e") file)))))
-             (t
-              (let ((cmd (run-hook-with-args-until-success
-                          'nim-compile-command-checker-functions file)))
-                (or cmd (nim--fmt nim-compile-default-command file))))))
+  ;; TODO set compile command for root project file
+  (when buffer-file-name
+    (let ((file (shell-quote-argument buffer-file-name)))
       (setq-local compile-command
-                  (if (or compilation-read-command current-prefix-arg)
-                      (compilation-read-command cmd)
-                    cmd)))))
+            (if
```

Comment:
Please check https://github.com/bbatsov/emacs-lisp-style-guide for proper `if`s style